tox.ini

```diff
@@ -65,28 +65,6 @@ commands =
     sphinx-apidoc -f -e -o docs/source/generated/ paasta_tools
     sphinx-build -b html -d docs/build/doctrees docs/source docs/build/html
 
-[testenv:paasta_itests]
```

Comment:
There seems to be a [testenv:paasta_itests_inside_container] environment too that sets `changedir=paasta_itests`, we can probably clean up that one too?